Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,68 @@
+﻿/*
+ * ********Lesson Notes for me here********
+ * Requirements for this bootcamp lesson(lesson 3)
+ *  1. Need to have refactored code to recieve inputs
+ *  2. Need to gather the following pieces of information:
+ *      -Name(string}
+ *      -Age(int)
+ *      -Height(Float)[This will be in Meters]
+ *      -Weight(Float)[This will be in Killograms]
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void RunLesson()
+        {
+            string name;
+            int age;
+            float personHeight,
+                  personWeight,
+                  bmi;
+
+            for (int i = 0; i < 3; i++)
+            {
+                name = GetString("What is your name?: ");
+                age = GetInteger("What is your age?: ");
+                personHeight = GetFloat("What is your height?: ");
+                personWeight = GetFloat("What is your weight?: ");
+
+                bmi = GetBMI(personHeight, personWeight);
```

Comment:
For GetBMI see the comment above. Also note that GetBMIFromPerson will give you a hard time so the convention is to name functions strictly in PascalCase -> GetBmiFromPerson.

Reply:
Changed GetBMI -> GetBmi as suggested